gradle.properties

```diff
@@ -175,7 +175,7 @@ grpcVersion=1.60.1
 
 guavaVersion=33.0.0-jre
 gwtVersion=2.10.0
-gwtServletVersion=2.11.0-jakarta-SNAPSHOT
+gwtServletVersion=2.11.0
```

Comment:
Where is this available? I'm not seeing it on mavenCentral.

Reply:
https://central.sonatype.com/artifact/org.gwtproject/gwt-servlet-jakarta/versions

Reply:
Ah, I see.  The coordinates are slightly different.

Reply:
Actually, can we update the property to gwtServletJakarta to represent the new coordinates?

Reply:
Sure

Reply:
BTW: What's the deal with `gwtVersion`? Is that property use hard-coded into the plugins or something? I wasn't able to update it along with servlet, but I didn't see where it was referenced.

Reply:
Yes, for better or for worse, the dependency on gwt-dev and gwt-user is set in our [Gwt gradle plugin](https://github.com/LabKey/gradlePlugin/blob/2b1db873f05ae2326585b67268b52e23ce7354dc/src/main/groovy/org/labkey/gradle/plugin/Gwt.groovy#L65).

Reply:
Lovely. Since you're more aware than I, can you recommend a way forward? Should I leave `gwtVersion` at `2.10.0`? Is there a new version I can set to match `gwtServletJakartaVersion`? Do we need to update the plugins?

Reply:
Okay, looks like it's hard-coded to look in `com.google.gwt`: https://central.sonatype.com/artifact/com.google.gwt/gwt-user/versions

But new coordinates are `org.gwtproject`: https://central.sonatype.com/artifact/org.gwtproject/gwt-user/versions

So plugins change?

Reply:
Old version (2.10.0) is also at the newer coordinates, so maybe we don't have to support both...